src/components/OpeningHour.jsx

```diff
@@ -8,33 +8,40 @@ const getStatusMessage = status => {
       color: '#60ad51',
     };
   }
+
   if (status === 'closed') {
     return {
       label: _('Closed'),
       color: '#8c0212',
     };
   }
+
   return { label: '', color: '#fff' };
 };
 
-const OpeningHour = ({ schedule }) => {
+const OpeningHour = ({ schedule, showNextOpenOnly = false }) => {
   if (!schedule) {
     return null;
   }
 
   const { isTwentyFourSeven, status, nextTransition } = schedule;
   const { label, color } = getStatusMessage(status);
-  if (isTwentyFourSeven) {
+
+  if (isTwentyFourSeven && !showNextOpenOnly) {
     return <div className="openingHour poi_panel__info__hours__24_7">
       {_('Open 24/7', 'hour block')}
       {' '}
       <div className="openingHour-circle" style={{ background: color }} />
     </div>;
   }
 
-  return <div className="openingHour">
+  return <div className="openingHour u-text--label">
     {label}
-    {nextTransition &&
+    {!showNextOpenOnly && nextTransition &&
```

Comment:
Looks like it can be simplified, with a single condition for this block and the one below.